### PR TITLE
Updates to jupyter notebook and ghtorrent

### PR DESCRIPTION
**Pull request by kmn5409**
I completed some examples using ghtorrent.py file and added that to the visualization in BaseRepoActivityCard and also added some more examples for the jupyter notebooks

### CODE REVIEW

**Comment by sgoggins** [comment]
@howderek : It looks like the failing issue is probably a rate limit on the GitHub API. Could you look into this so we know what's up with the failing dev build. I don't think its just this pull request.

File: augur/server.py [hunk 1]
```diff
@@ -16,24 +16,14 @@
...
-CONFIG_BAD = False
+
+
 '''
 make a try and accept condition
 if its open the GH_DATA_ONFIG_FILE and then its open in read mode
 and if the file does't open the it print Couldn\'t open config file, attempting to create.
 '''
 
-try:
```
Comment (sgoggins) [inline augur/server.py]: @howderek : Maybe I am reading this wrong, but I don't think we want to delete this stuff from server.py ...

File: augur/server.py [hunk 2]
```diff
@@ -83,72 +73,32 @@ def addTimeseries(app, function, endpoint):
     app.route('/{}/<owner>/<repo>/timeseries/{}/relative_to/<ownerRelativeTo>/<repoRelativeTo>'.format(AUGUR_API_VERSION, endpoint))(flaskify(augur.util.makeRelative(function)))
 
 
+# Create Flask application
 app = Flask(__name__)
-CORS(app)# Try to open the config file and parse it
-parser = configparser.RawConfigParser()
```
Comment (sgoggins) [inline augur/server.py]: @howderek Are there changes we made in the past few days not reflected in @kmn5409 's commits ... this doesn't look right ... to delete this config checking stuff.

**Comment by howderek** [comment]
Hi @kmn5409, I cannot accept these changes

File: frontend/app/components/BaseRepoActivityCard.vue [hunk 3]
```diff
@@ -7,16 +7,16 @@
...
-                    cite-text="Community Activty">
+                    cite-url="https://github.com/chaoss/metrics/blob/master/activity-metrics/code-commits.md"
+                    cite-text="Commits">
         </line-chart>
       </div>
 
       <div class="col col-6">
         <line-chart source="forks"
                     title="Forks / Week" 
-                    cite-url="https://github.com/chaoss/metrics/blob/master/activity-metrics/community-activity.md"
```
Comment (sgoggins) [inline frontend/app/components/BaseRepoActivityCard.vue]: This is a sample code review comment.

**Pull request closed without merging**